src/helpers/providers/relayProvider.ts

```diff
@@ -1,27 +1,9 @@
 import { Eip1193Bridge } from '@ethersproject/experimental'
-import { PingFilter } from '@opengsn/common'
 import { RelayProvider } from '@opengsn/provider'
 import { Web3Provider } from '@ethersproject/providers'
 import { WrapBridge } from '@opengsn/provider/dist/WrapContract'
 import env from 'helpers/env'
 
-const GasPricePingFilter: PingFilter = (
```

Comment:
I don't think that we need to remove this, all of my local test was successful with this filter